metrics/mixin/alerts/mds.libsonnet

```diff
@@ -0,0 +1,26 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'mds-memory-usage-alerts.rules',
+        rules: [
+          {
+            alert: 'MDSCacheUsage',
+            expr: |||
+              ceph_mds_mem_rss / on(ceph_daemon) group_left(job)(label_replace(kube_pod_container_resource_requests{container="mds", resource="memory"}, "ceph_daemon", "mds.$1", "pod", "rook-ceph-mds-(.*)-(.*)")*.5) > 1
```

Comment:
@sp98 , is it possible to give an out put of

`ceph_mds_mem_rss / on(ceph_daemon) group_left(job)(label_replace(kube_pod_container_resource_requests{container="mds", resource="memory"}, "ceph_daemon", "mds.$1", "pod", "rook-ceph-mds-(.*)-(.*)")*.5)`

without any condition, just to verify we have the output, of the expression, as expected

Reply:
I don't have a live cluster currently. Give me sometime. I'll past the output here.